commit bd22f7d5cb176f68424d7d2463df34d19ffaf09c
Author: Sven Joachim <svenjoac@gmx.de>
Date:   Sun Sep 1 19:38:48 2013 +0200

    Don't build the eglfbdev demo
    
    It does not compile on non-Linux architectures and is not installed
    anyway.

---
 src/egl/opengles1/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

--- a/src/egl/opengles1/Makefile.am
+++ b/src/egl/opengles1/Makefile.am
@@ -42,7 +42,6 @@
 	bindtex \
 	clear \
 	drawtex_x11 \
-	eglfbdev \
 	es1_info \
 	gears_x11 \
 	msaa \
